analysis/tests/src/expected/CompletionFromModule.res.txt

```diff
@@ -0,0 +1,79 @@
+
+Complete src/CompletionFromModule.res 11:5
+posCursor:[11:5] posNoWhite:[11:4] Found expr:[11:3->11:5]
+Pexp_field [11:3->11:4] _:[15:0->11:5]
+[set_result] set new result to Cpath Value[n].""
+Completable: Cpath Value[n].""
+Package opens Pervasives.JsxModules.place holder
+Resolved opens 1 pervasives
+ContextPath Value[n].""
+[ctx_path]--> CPField
+ContextPath Value[n]
+[ctx_path]--> CPId
+Path n
+Path SomeModule.
+[{
+    "label": "->SomeModule.getName",
```

Comment:
curious about the function coming before the field
also, any thoughts about making `.` a valid source language construct?

Reply:
Could you expand a bit?

Reply:
1 If I write `x.` I see `s.getName()` before `x.field`, which can be surprising to people working with records. Just wondering.

2 Could `x.getName()` be actually part of the language, or will it always need to be expressed as `x->getName`? Autocompletion is right now suggestive of something that does not exist in the language. Not necessarily a bad thing, just thinking aloud about possible pros and cons here.

Reply:
1. Ahh right! That's a good point. It's currently sorted on the field name and function name. We could sort it whatever way we think works best. I'd say either this (sort it together with the regular fields) or put it at the bottom. I'd favor the current way, I think.
2. Oh, that's.... a very interesting idea! That would solve quite a few of our issues. Let's explore that!

Reply:
Thank you for the context. Will read through.

Reply:
I wonder if we could figure out a good way to do that, pointing out the module to use with the call (`someRecord.SomeModule.someFn()`), but with a small enough twist that it's still more convenient than pipe, but not ambigious wrt the current behavior of `SomeModule` there being a hint.

Anyway, just completing the actual pipe, but via dot, is a good start for DX imo.